packages/react/src/details/Details.tsx

```diff
@@ -42,9 +42,11 @@ export const Details = forwardRef<HTMLDetailsElement, DetailsProps>(
 				>
 					{iconBefore && (
 						<InfoIcon
-							weight="regular"
-							size="md"
+							aria-hidden={false}
+							aria-label="Information."
```

Comment:
I wonder if we should remove the full-stop to make it aligned with our alert icons?

Reply:
Hmm maybe. It read so much cleaner this way, though. Especially when the label is a question.

`Information how were my details prefilled?`

vs

`Information. How were my details prefilled?`

Reply:
Yeah, I agree it probably reads nicer. Happy to leave it in and take it to the next round of testing